doc/developer-guide/deployment/https.rst

```diff
@@ -3,9 +3,10 @@
 HTTPS support
 =============
 
-Zotonic has built-in support for https and certificate handling.
+Zotonic has built-in support for HTTPS and TLS (previously SSL) certificate
+handling.
 
-All sites share a single ssl port using Server Name Indication for 
+All sites share a single TLS port using Server Name Indication for
```

Comment:
I agree that technically we should use TLS.

The problem @jult mentions is that everybody, everywhere, is using SSL.  As in "Buy here your SSL Certificates".  See for example https://www.digicert.com/